rpc/handler.go

```diff
@@ -64,28 +64,30 @@ type handler struct {
 	subLock    sync.Mutex
 	serverSubs map[ID]*Subscription
 
-	executionPool *SafePool
+	executionPoolRequestTimeout time.Duration
+	executionPool               *SafePool
```

Comment:
If we don't have request specific timeouts, then `executionPoolRequestTimeout` can be part of executionPool.

Reply:
also it should be atomic to make it changeable in runtime